src/bartender/__main__.py

```diff
@@ -51,6 +54,26 @@ def make_super(email: str) -> None:
     asyncio.run(make_super_async(email))
 
 
+def mix(config: Path) -> None:
+    """Runs arq worker to run queued jobs.
+
+    Like a bartender mixes the cocktails,
+    the i-vresse bartender mixes aka runs queued jobs.
+
+    :param config: Path to config file.
+    :raises ValueError: When no usefull destination found in config file.
+    """
+    validated_config = build_config(config)
+    configs: list[ArqSchedulerConfig] = []
+    for destination in validated_config.destinations.values():
+        if isinstance(destination.scheduler, ArqSchedulerConfig):
+            configs.append(destination.scheduler)
+    if not configs:
+        raise ValueError("No destination found in config file using arq scheduler")
+
+    asyncio.run(run_workers(configs))
```

Comment:
It would be nice to get some cli message that the worker is started and waiting for jobs, or something like that

Reply:
print added